app/src/main/java/com/amaze/filemanager/utils/Utils.java

```diff
@@ -263,4 +263,32 @@ public static String formatTimer(long timerInSeconds) {
         final long sec = TimeUnit.SECONDS.toSeconds(timerInSeconds - TimeUnit.MINUTES.toSeconds(min));
         return String.format("%02d:%02d", min, sec);
     }
+
+    public static String wildcardToRegex(String wildcard) {
```

Comment:
Why is this method needed?

Reply:
Convenience method for package name checking, so `FtpReceiver.blacklistedApps` can use wildcards to define package names to blacklist.